src/lib/pages/backoffice/DocumentRequest/DocumentRequestEditor/uiSchema.js

```diff
@@ -49,7 +49,8 @@ export const uiSchema = (title) => {
         'custom:divider': 16,
       },
       {
-        state: 8,
+        state: 4,
+        request_type: 4,
```

Comment:
Careful with this :) I think I did not add it on purpose.
The workflow of a document request varies depending on its state. Changing its state without knowing how to change other metadata can be risky.
The state of a Document Request can be changed via actions in the backoffice UI and it should be changed via the form, because it is basically `Do it only if you know what you are doing, at your own risk`.
We allow to change the state in the form only if it does **not** affect the workflow or the state of other records.

Reply:
ah ok! This should only be a simple switch of places.

Reply:
Oups! I am sorry, I didn't see it... my mistake...